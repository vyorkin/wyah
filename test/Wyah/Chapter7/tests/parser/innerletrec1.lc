let f = \x -> (let rec y = \z -> z in y);
