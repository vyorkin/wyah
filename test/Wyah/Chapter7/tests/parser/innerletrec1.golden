Program [Decl "f" (ELam (Var {unVar = "x"}) (ELet (Var {unVar = "y"}) (ELam (Var {unVar = "z"}) (EVar (Var {unVar = "z"}))) (EVar (Var {unVar = "y"}))))]